### PR TITLE
make config-store identifier key name unique

### PR DESCRIPTION
**Pull request by tomekl007**

### CODE REVIEW

File: config-store-yaml/src/main/java/io/stargate/config/store/yaml/ConfigStoreActivator.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ private synchronized void startConfigStore(Metrics metrics) {
     started = true;
 
     Hashtable<String, String> props = new Hashtable<>();
-    props.put("Identifier", CONFIG_STORE_YAML_IDENTIFIER);
+    props.put("ConfigStoreIdentifier", CONFIG_STORE_YAML_IDENTIFIER);
```
Comment (dimas-b) [inline config-store-yaml/src/main/java/io/stargate/config/store/yaml/ConfigStoreActivator.java:82]: Just wondering: What is the reason for this change?

**Review by dimas-b — APPROVED** [review]

File: config-store-yaml/src/main/java/io/stargate/config/store/yaml/ConfigStoreActivator.java (hunk 1 above)
Comment (tomekl007) [reply config-store-yaml/src/main/java/io/stargate/config/store/yaml/ConfigStoreActivator.java:82]: I see that we have this pattern for authIdentifier (having unique selector name) 
It maybe easier to find usages of it instead of using Identifier name everywhere.
